### PR TITLE
Update tigera-ca-bundle path to /etc/pki/tigera

### PR DESCRIPTION
**Pull request by vara2504**
## Description

Must be merged with https://github.com/tigera/fluentd-docker/pull/295

Updating the tigera-ca-bundle path as suggested in https://tigera.slack.com/archives/C03QHUBTEGP/p1668014386298799
from /tigera-ca-bundle/ to /etc/pki/tls/certs/

Background:
While mounting on "/etc/pki/tls/certs/" existing Internet ca-bundle.crt which was bundle as part of [tigera fluentd docker image ](https://github.com/tigera/fluentd-docker/blob/master/Dockerfile#L29)is overwritten.

Hence replacing all the tigera-ca-bundle reference to /etc/pki/tigera/

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: pkg/render/manager_test.go
Comment (vara2504) [reply pkg/render/manager_test.go]: Updated

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
